demo/models/config.yml

```diff
@@ -0,0 +1,18 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+block_id: <block-id>
+name: ModuleName
+version: 0.0.1
+artifacts_path: /absolute/path/to/model/artifacts
```

Comment:
this is okay but it can be relative or absolute, so seems like if this example suggests it has to be absolute